crates/subspace-node/src/domain/auto_id_chain_spec.rs

```diff
@@ -172,6 +172,9 @@ fn get_operator_params(
     }
 }
 
+/// Returns AutoId genesis domain.
+/// Note: Currently unused since dev or devnet uses EVM domain and not AutoId
+#[allow(dead_code)]
```

Comment:
If this code is always unused, you could mark it as:
```suggestion
#[expect(dead_code)]
```

Then if we use it in future, we'll get a compile error and remove that annotation.